src/hotspot/share/gc/g1/g1CardSetMemory.hpp

```diff
@@ -84,44 +84,30 @@ typedef G1SegmentedArrayFreeList<mtGCCardSet> G1CardSetFreeList;
 // Since it is expected that every CardSet (and in extension each region) has its
 // own set of allocators, there is intentionally no padding between them to save
 // memory.
-template <class Slot>
 class G1CardSetAllocator {
-  // G1CardSetSegment management.
-
-  typedef G1SegmentedArray<Slot, mtGCCardSet> SegmentedArray;
   // G1CardSetContainer slot management within the G1CardSetSegments allocated
   // by this allocator.
-
-  SegmentedArray _segmented_array;
+  G1SegmentedArray<mtGCCardSet> _segmented_array;
   FreeListAllocator _free_slots_list;
 
 public:
   G1CardSetAllocator(const char* name,
-                     const G1CardSetAllocOptions* alloc_options,
-                     G1CardSetFreeList* free_segment_list);
+                         const G1CardSetAllocOptions* alloc_options,
+                         G1CardSetFreeList* free_segment_list);
```

Comment:
I think the indentation shouldn't be changed here - the parameters should be aligned.